src/package/package.js

```diff
@@ -93,21 +74,24 @@ async function setPackagesList(instPath, minorUpdate = false) {
     'タイプ',
     '最新バージョン',
     '現在バージョン',
+    '解説',
+    'リンク',
   ];
   const packages = await getPackages(instPath);
 
-  // table header
-  if (!thead.hasChildNodes()) {
-    const headerTr = document.createElement('tr');
-    for (const [i, columnName] of columns.entries()) {
-      const th = document.createElement('th');
-      th.classList.add('sort');
-      th.setAttribute('data-sort', columnName);
-      th.setAttribute('scope', 'col');
-      th.innerText = columnsDisp[i];
-      headerTr.appendChild(th);
-    }
-    thead.appendChild(headerTr);
+  // sort buttons
+  if (!packagesSort.hasChildNodes()) {
+    Array.from(columns.entries())
+      .filter(([n, s]) => ['name', 'developer'].includes(s))
+      .forEach(([i, columnName]) => {
+        const sortBtn = document
+          .getElementById('sort-template')
+          .cloneNode(true);
+        sortBtn.removeAttribute('id');
+        sortBtn.setAttribute('data-sort', columnName);
```

Comment:
And while we're at it, let's change this, too.
```suggestion
        sortBtn.dataset.sort = columnName;
```